stockflux-securities-master/src/components/ButtonAddSecurity.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Button from "./Button";
+
+const ButtonAddSecurity = ({ size }) => {
```

Comment:
I think this would be better named `AddSecurityButton`. I'm not sure it's worth having this as a separate component either since it's just a simple button and used only once. There are other places where you've got a `<Link>` wrapping a `<Button>` as well.

Reply:
removed this now